server/src/test/java/org/elasticsearch/index/engine/LuceneChangesSnapshotTests.java

```diff
@@ -272,4 +310,8 @@ public void run() {
         }
         return operations;
     }
+
+    private int randomSearchBatchSize() {
```

Comment:
we only need a method for this because the default is long. Can we use a short default name or a constant and just inline this.